README.md

```diff
@@ -1,106 +1,27 @@
-# SwiftSyntax
+# Swift Syntax
 
-SwiftSyntax is a set of Swift libraries for parsing, inspecting, generating, and transforming Swift source code.
+The swift-syntax package is a set of libraries that work on a source-accurate tree representation of Swift source code, called the SwiftSyntax tree. The SwiftSyntax tree forms the backbone of Swift’s macro system – the macro expansion nodes are represented as SwiftSyntax nodes and a macro generates a SwiftSyntax tree to be inserted into the source file.
 
 ## Documentation
 
 You can read SwiftSyntax’s documentation on [swiftpackageindex.com](https://swiftpackageindex.com/apple/swift-syntax/documentation).
 
 A great way to interactively explore the SwiftSyntax tree of a source file is https://swift-ast-explorer.com, developed by [@kishikawakatsumi](https://github.com/kishikawakatsumi).
 
-## Adding SwiftSyntax as a Dependency
+A set of example usages of swift-syntax can be found in [Examples](Examples).
 
-### Trunk Development (main)
+## Releases
 
-The mainline branch of SwiftSyntax tracks the latest developments. It is not
-an official release, and is subject to rapid changes in APIs and behaviors. To 
-use it, add this repository to the `Package.swift` manifest of your project:
+Releases of SwiftSyntax are aligned with corresponding language and tooling releases, for example the major version 509 of swift-syntax is aligned with Swift 5.9.
 
-```swift
-// swift-tools-version:5.7
-import PackageDescription
-
-let package = Package(
-  name: "MyTool",
-  dependencies: [
-    .package(url: "https://github.com/apple/swift-syntax.git", branch: "main"),
-  ],
-  targets: [
-    .target(name: "MyTool", dependencies: [
-      .product(name: "SwiftSyntax", package: "swift-syntax"),
-    ]),
-  ]
-)
-```
```

Comment:
Personally, I would opt for keeping a basic 'Installation' or 'Getting Started' section. I find it very useful, even just for copying the dependency code.

Maybe something like this:
 
<img width="867" alt="Screenshot 2023-09-18 at 8 58 33 PM" src="https://github.com/apple/swift-syntax/assets/44930823/7728f405-3415-47fc-b59d-956debcc4939">

or

<img width="887" alt="Screenshot 2023-09-18 at 9 00 08 PM" src="https://github.com/apple/swift-syntax/assets/44930823/92e7cb0c-0918-47a7-8575-1a4cfd35b559">

?

Reply:
Makes sense as a copy-paste-template. I added it back in.